src/containers/myEvents/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Radio, Typography } from 'antd';
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { connect, useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { ChungusContentContainer } from '../../components';
+import Calendar from '../../components/Calendar';
+import EventsList from '../../components/events-list/EventsList';
+import { C4CState } from '../../store';
+import { asyncRequestIsComplete } from '../../utils/asyncRequest';
+import { getMyEvents } from './ducks/thunks';
+import { MyEventsReducerState } from './ducks/types';
+const { Title } = Typography;
+
+const Content = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const StyledRadio = styled(Radio.Group)`
+  margin: 12px;
+`;
+
+const StyledTitle = styled(Title)`
+  margin: 12px;
+  margin-left: 0px;
+`;
+
+interface MyEventsProps {
```

Comment:
This is a shadowed name, but it points to a larger issue unrelated to this PR specifically. 

We need to rename the `EventsProps` and `MyEventsProps` interfaces. We should only use `Props` in an interface when it is directly for a component, and only used there. 

In this case, I think `EventDetails` and `MyEventDetails` are appropriate, with `MyEventDetails` extending `EventDetails`. 

It's fine to use these interfaces as prop types for components, but the "source of truth" so to say should be dictated by a `ducks/types.ts` file, not the component-level needs. 

cc @rymaju

Reply:
I believe there's already a component called EventDetails which would cause some conflict with imports. Will name something along the lines of EventInformation/MyEventInformation